amethyst_core/Cargo.toml

```diff
@@ -25,8 +25,8 @@ log = "0.4.6"
 num-traits = "0.2.0"
 rayon = "1.1.0"
 serde = { version = "1", features = ["derive"] }
-specs = { version = "0.15", features = ["shred-derive"] }
-specs-hierarchy = "0.5.1"
+specs = { version = "0.16", features = ["shred-derive", "storage-event-control"] }
```

Comment:
hm, could you enable the `"specs-derive"` feature by default on `specs`, and for `"storage-event-control"`, have that as a feature of `amethyst` and `amethyst_core`, which, when enabling it on `amethyst, it wires through to enable it on `specs`?